### PR TITLE
Server-side rendering

### LINKED ISSUE #140: Server-side rendering
Rewrite the frontend pages to make use of SSR.

### PR DESCRIPTION
**Pull request by cjdenio**
Closes #140

### CODE REVIEW

File: frontend/layouts/app.tsx [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
-  const { data: team } = useSWR(() => `/teams/${app.app.TeamID}`);
-
+  app,
+  user,
+  team,
+}: PropsWithChildren<{
+  selected: string;
+  app: IApp | undefined;
+  user: IUser | undefined;
+  team: ITeam | undefined;
```
Comment (anirudhb) [inline frontend/layouts/app.tsx]: `app?: IApp`

File: frontend/layouts/dashboard.tsx [hunk 2]
```diff
@@ -144,19 +145,15 @@ export default function DashboardLayout({
   image,
   sidebarSections,
   children,
+  user,
 }: PropsWithChildren<{
   title: string;
   image?: string;
   sidebarSections: ISidebarSection[];
+  user: IUser | undefined;
```
Comment (anirudhb) [inline frontend/layouts/dashboard.tsx]: `user?: IUser`

File: frontend/pages/teams/[id].tsx [hunk 3]
```diff
@@ -305,3 +316,35 @@ export default function TeamPage() {
...
 }
+
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
+  try {
+    const [user, team] = await Promise.all(
+      ["/users/me", `/teams/${ctx.params.id}`].map((i) => fetchSSR(i, ctx))
+    );
+
+    if (team.team.Personal) {
+      throw { message: "can't visit personal team" };
```
Comment (anirudhb) [inline frontend/pages/teams/[id].tsx:327]: Redirect to dashboard instead

File: pkg/biller/biller.go [hunk 4]
```diff
@@ -74,7 +74,7 @@ func StartBillingApp(conn *dokku.DokkuConn, app db.App) error {
 	// Get the app's container ID
 	cid, err := conn.RunCommand(context.Background(), []string{"haas:cid", app.ShortName})
 	if err != nil {
-		return err
+		return nil
```
Comment (anirudhb) [inline pkg/biller/biller.go]: why

**Review by anirudhb — CHANGES_REQUESTED** [review]
lgtm

File: frontend/pages/teams/[id].tsx (hunk 3 above)
Comment (cjdenio) [reply frontend/pages/teams/[id].tsx:327]: I was thinking a 404 would make more sense (that's what this does)

File: pkg/biller/biller.go (hunk 4 above)
Comment (cjdenio) [reply pkg/biller/biller.go]: Oops, that was a temporary fix to make my dev environment work; sorry 😬

**Review by anirudhb — APPROVED** [review]

File: frontend/pages/teams/[id].tsx (hunk 3 above)
Comment (anirudhb) [reply frontend/pages/teams/[id].tsx:327]: Nooo pls make this redirect to dashboard

Comment (anirudhb) [reply frontend/pages/teams/[id].tsx:327]: Ahhh it's already merged but i was requesting changes on that
Never turn on auto merge for any PR other than dependabot :(

**Pull request merged**